openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -71,7 +71,16 @@
     line-height: 0;
   }
 
-
+  // submission answer leaderboard fix (TNL-4009). 4 class selectors is more specific than
+  // lms/static/sass/course/courseware/_courseware.scss#L453 in edx-platform (3 class selectors)
+  .leaderboard__answer .submission__answer__display__content {
+    h2.submission__answer__part__text__title {
+      @include text-align(left);
+      @include float(none);
+      font-size: inherit;
+      margin: 0;
+    }
+  }
```

Comment:
If `_shame.scss` isn't the right place for this code, let me know. This case seems to fit "rules to overcome any poorly defined, scoped, or architected edX LMS-based CSS" as described above.